cfme/tests/cli/test_appliance_console.py

```diff
@@ -760,7 +759,30 @@ def test_appliance_console_restart():
             9.
             10. Confirm that Appliance is running into browser.
     """
-    pass
+    cmd_stop_EVM = ("ap", RETURN, "16", "Y", RETURN, RETURN, "21")
+    appliance.appliance_console.run_commands(cmd_stop_EVM, timeout=90)
+    # Checking EVM service stoppped or not
+    wait_for(
+        lambda: not appliance.is_web_ui_running(),
+        delay=5,
+        num_sec=60,
+        message=f"Waiting to stop EVM service"
+    )
+
+    cmd_start_EVM = ("ap", RETURN, "17", "Y", RETURN, RETURN, "21")
+    appliance.appliance_console.run_commands(cmd_start_EVM, timeout=90)
+    # Checking EVM service started or not
+    wait_for(
+        lambda: appliance.is_web_ui_running(),
```

Comment:
Same as above. Simply using `wait_for(appliance.is_web_ui_running, ....)` would do the trick.